bmlt-meeting-list.php

```diff
@@ -26,36 +26,71 @@ class Bread {
 		var $formats_all = '';
 		
 		var $service_meeting_result ='';
-		var $optionsName = 'bmlt_meeting_list_options';
+		const SETTINGS = 'bmlt_meeting_list_settings';
+		const OPTIONS_NAME = 'bmlt_meeting_list_options';
+		var $optionsName = Bread::OPTIONS_NAME;
 		var $options = array();
+		var $outside_meeting_result = array();
+		var $allSettings = array();
+		var $maxSetting = 1;
+		var $loaded_setting = 1;
+		
+		function loadAllSettings() {
+		    $this->allSettings = get_option( Bread::SETTINGS );
```

Comment:
why not have the function return a default setting array instead of running through this double check?  encapsulate the functionality?

Reply:
This is supposed to make the upgrade case work.  That is, the user already has a defined meeting list (in options) but does not have any settings (because older versions of bread dont have this).  In this case we need to fake a setting for him.  If he's already upgrade, then we go to the else clause.  Does that answer your question?

Reply:
I understand that, I meant to have `get_option()` function always return the correct array.